app/models/edm/web_resource.rb

```diff
@@ -41,6 +41,7 @@ class WebResource
     validates :media, presence: true, if: :published?
     validates :edm_rights, presence: true, unless: :media_blank?
     validate :europeana_supported_media_mime_type, unless: :media_blank?
+    validate :media_size, unless: :media_blank?
```

Comment:
We need to make `:media_blank?` more efficient. Every time it is called, it results in a request to the underlying object storage service. Therefore, during validation of a web resource, as of this branch, three identical requests are made. At most one should be made during validation.

Reply:
checking for `media.identifier.nil?` seems to work